circle.yml

```diff
@@ -10,10 +10,19 @@ dependencies:
   cache_directories:
   - .sbt-launch.jar
   override:
-  - ./sbt update test:compile e2e:compile integration:compile
+  - >
+    if [ -n "${TWITTER_DEVELOP}" ]; then
+      ./project/install-twitter-develop-deps.sh
+      ./sbt 'set developTwitterDeps in Global := true' update
+    else
+      ./sbt update
+    fi
 
 test:
   override:
   # unfortunately these have to be two separate sbt commands ¯\_(ツ)_/¯
   - ./sbt coverage test && ./sbt coverageAggregate
-  - ./sbt e2e:test assembly
+  - ./sbt integration:compile e2e:test assembly
```

Comment:
I couldn't easily figure this out so will just ask -- do these sbt commands not also require `'set developTwitterDeps in Global := true'` in order to run against the finagle snapshots?